src/batch/Cargo.toml

```diff
@@ -49,7 +49,7 @@ tokio = { version = "1", features = [
     "fs",
 ] }
 tokio-stream = "0.1"
-tonic = "0.7"
+tonic = { version = "0.2.0-alpha.1", package = "madsim-tonic" }
```

Comment:
We will need to patch tonic? Otherwise some packages using tonic underlying will still use the upstream ones.

By the way, is there any possibility of switching tonic package based on feature flags? Or can we `cat "[patch.tonic]\nxxxx" >> Cargo.toml` on CI when needed, so that developers can still use the upstream tonic?

Reply:
Same question

Reply:
Oh, that's a good point. Seems that we need to use the patch. But cargo requires that the patched package should have the same version as upstream. It's not easy to make it work. 🙃

Reply:
Is there any package depending on tonic now?

Reply:
Maybe we can use cargo tree to take a look?

Reply:
> Is there any package depending on tonic now?

Only one `etcd-client`

Reply:
Okay, that's only used on meta service, and currently all test cases you modified seem to be part of compute node.